### DISCUSSION TITLE
Obsidian Sync setup

### DISCUSSION THREAD

**Discussion by gavinmcfall** [MobiusSync/MobiusSync#226] [Q&A]
![image](https://github.com/user-attachments/assets/effb3d07-41d7-4624-8312-02431ed84c40)

I get three errors when trying this on the iPad

- Loading ignores: lstat /dev/null/inaccessible/.stignore: not a directory
- Failed to create folder root directory: stat /dev/null/inaccessible/.stignore: not a directory
- Error on folder "Obsidian" (Obsidian) at bookmark:Obsidian: stat /dev/null/inaccessible/.stignore: not a directory

My Process was:

1. Open Obsidian on the ipad and create a temporary vault so the folder structure is created
2. Go onto my home PC, into syncthing and share the folder with the iPad
3. Then on the iPad, accept the share
4. Choose external folder
5. navigate to the local folder for obsidian on the ipad and choose it

NOTE: I did the exact same steps on my iPhone and it works just fine. This is unique to the iPad only

iPhone 15 pro: iOS 18.1.1
iPad Pro 12.9": iPad OS 18.1.1

**Comment by trzpiot** [comment]
Unfortunately, I have the same problem, but with an iPhone 13 (iOS 18.2).

**Comment by neilbaldwin** [comment]
Same problem here - iPad 11 Pro, iOS 18.1.1

I'm sure I used to have this working but I hadn't touched it for a while and decided to start afresh.

To me the issue is there is no option to open an existing vault on iOS, you only have the option to create one. This has been the case for a couple of years now.

**Comment by neilbaldwin** [comment]
After hacking around with this for some time I think I've figured out the exact steps needed:

This assumes you have Syncthing running on your computer and you have an existing Obsidian Vault on your computer. Also assumes you have a clean installation of Obsidian on your iPad.

1) Open Obsidian app on iPad and create a new vault with a temporary name, say, “temp”. Uncheck the “Store in iCloud” option. This will create a folder structure on your iPad. In Files app it will be “Obsidian/temp”

2) In the Mobius app, select Add Folder then select “Pick External Folder”. Important: navigate out of the Mobius folder if you are in it and find the Obsidian folder you just created. Ignore the “temp” folder and create another folder here called, say, “Vault”. You’ll get a warning about the folder being outside of the Mobius sandbox but that’s OK.

3) You should now have a new folder in Mobius called “Vault” that is unshared. Tap on this folder and then select Edit, then the Sharing tab. Tick the box to share the folder to your computer.

4) On your computer (Syncthing web app) you should get a request to share the “Vault” folder from your iPad. Click “Add". Now in the Folder Path box, change the path to be the same as your Synthing/Obsidian folder that you want to use to sync Obsidian between the two devices. For example, `/Users/yourname/Syncthing/Obsidian` Then click Save.

5) The folder should sync between your two devices. Back on your iPad the status of the folder in Mobius should say Up to Date in green.

6) Now open up Obsidian on your iPad. Open the sidebar and at the top you should still see “temp” as the open vault. Tap on that then tap on “Manage Vaults”. You should now see your new vault “Vault” in the list. Tap to open the synced vault then tap “Enter Vault” on the next page.

Hope that helps some others who are struggling with this. Merry Christmas!

**Comment by SimonPickup** [comment]
> ![image](https://private-user-images.githubusercontent.com/131718663/392129110-effb3d07-41d7-4624-8312-02431ed84c40.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyMzI1NDUsIm5iZiI6MTczNTIzMjI0NSwicGF0aCI6Ii8xMzE3MTg2NjMvMzkyMTI5MTEwLWVmZmIzZDA3LTQxZDctNDYyNC04MzEyLTAyNDMxZWQ4NGM0MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNlQxNjU3MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNmM0ODY4M2VjMTJhNzlmZTY0MWZmOGJjODBiYmM0NDFjM2M4MjNhNzA4NmVkMTdkMGZjMmYxMGQwZmM0YTlhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.5UQbSTfnS1jRk1ulFBTLfQCKI2M2XcH8GhR1O6sqhWk)
> 
> I get three errors when trying this on the iPad
> 
>     * Loading ignores: lstat /dev/null/inaccessible/.stignore: not a directory
> 
>     * Failed to create folder root directory: stat /dev/null/inaccessible/.stignore: not a directory
> 
>     * Error on folder "Obsidian" (Obsidian) at bookmark:Obsidian: stat /dev/null/inaccessible/.stignore: not a directory
> 
> 
> My Process was:
> 
>     1. Open Obsidian on the ipad and create a temporary vault so the folder structure is created
> 
>     2. Go onto my home PC, into syncthing and share the folder with the iPad
> 
>     3. Then on the iPad, accept the share
> 
>     4. Choose external folder
> 
>     5. navigate to the local folder for obsidian on the ipad and choose it
> 
> 
> NOTE: I did the exact same steps on my iPhone and it works just fine. This is unique to the iPad only
> 
> iPhone 15 pro: iOS 18.1.1 iPad Pro 12.9": iPad OS 18.1.1

Sorry for the delay on this - I have had a death in the family.

I cannot get it to fail in the way you indicate.

Please advise which variant and version you are using. I am guessing a fairly recent TestFlight.

Are you trying to sync the whole Obsidian folder or only one vault's folder?

I notice your bookmark is named "bookmark:Obsidian", which is because you have changed the folder name or ID Can you try with defaults?

Do the set of steps explained by @neilbaldwin work for you?

**Reply by DilpreetSinghh** [reply]
Yes I have followed those steps but sync says
```
Loading ignores: lstat /dev/null/inaccessible/.stignore: not a directory
Failed to create folder root directory: stat /dev/null/inaccessible/.stignore: not a directory
Error on folder "Obsidian" (Obsidian) at bookmark:Obsidian: stat /dev/null/inaccessible/.stignore: not a directory
```

Then I have 
- update iPad Os to 18.4
- Mobius Sync Pro to 1.28.0

It started to sync but now


![Screenshot 2025-04-02 at 11 37 43](https://github.com/user-attachments/assets/d3572cd9-6bd4-4f84-8232-f0ef04b95755)

- 2025-04-02 11:35:55: Failure checking encryption consistency with device Dilpreet Android for folder "Obsidian" (3hgbs-gewfy): remote expects to exchange plain data, but local data is encrypted (folder-type receive-encrypted)

- 2025-04-02 11:35:57: Failure checking encryption consistency with device Dilpreet Android for folder "Obsidian" (3hgbs-gewfy): remote expects to exchange plain data, but local data is encrypted (folder-type receive-encrypted)

- 2025-04-02 11:36:36: Failure checking encryption consistency with device Dilpreet Android for folder "Obsidian" (3hgbs-gewfy): remote expects to exchange plain data, but local data is encrypted (folder-type receive-encrypted)

**Reply by DilpreetSinghh** [reply]
I know it might seems that I have turned on encryption from one device and another is unencrypted, or Folder type is `Received Encrypted` instead of `Receive only` or Send and Receive`.

It's not that case.

**Reply by DilpreetSinghh** [reply]
issue is solved, but I have to give up encryption on iOS and iPadOS. after syncing if I turn on encryption. it says `Out of Sync` on new edits.

```
2025-04-02 19:43:35 Puller (folder "Obsidian" (3hgbs-gewfy), item "Notes/Rough.md"): syncing: finishing: pull: generic error
2025-04-02 19:43:35 "Obsidian" (3hgbs-gewfy): Failed to sync 1 items
2025-04-02 19:43:35 Folder "Obsidian" (3hgbs-gewfy) isn't making sync progress - retrying in 4m1s.
```

any help?

**Comment by gavinmcfall** [comment]
For what ever reason, following the same original steps this morning. Worked. Not sure if something in iPad OS, Mobius or Obsidian was updated, but no errors and syncing is working

**Reply by DilpreetSinghh** [reply]
Does your setup include encryption of data while sharing?
